frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -1,74 +1,84 @@
 import { Button, Container, Nav, Navbar, NavDropdown } from "react-bootstrap";
 import { Link } from "react-router-dom";
-import { hasRole} from "main/utils/currentUser";
+import { hasRole } from "main/utils/currentUser";
+import AppNavbarLocalhost from "main/components/Nav/AppNavbarLocalhost"
 
-export default function AppNavbar({currentUser, systemInfo, doLogout}) { 
+export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUrl = window.location.href }) {
+  // const url = currentUrl || window?.location?.href;
   return (
-    <Navbar expand="xl" variant="dark" bg="dark" sticky="top">
-      <Container>
-        <Navbar.Brand as={Link} to="/">
-          Example
-        </Navbar.Brand>
+    <>
+      {
+        (currentUrl.startsWith("http://localhost:3000") ||
+          currentUrl.startsWith("http://127.0.0.1:3000")) && (
+          <AppNavbarLocalhost url={currentUrl} />
```

Comment:
Create React App should set `NODE_ENV` to `development` when running in development mode - can we check that in addition to checking URL? Might be safe to do so because I believe Heroku randomly assigns ports to its dynos and there's a _slight_ chance it could pick 3000 and you would get this message in production.

Reply:
@btk5h is correct, and we are really behind, so I'd like to defer the NODE_ENV check to a future issue.

Reply:
Incoporated into issue #60 since this could take a while and we are on a tight deadline.